runtime/lua/vim/lsp/semantic_tokens.lua

```diff
@@ -580,7 +591,25 @@ function STHighlighter:reset_timer()
   end
 end
 
-local M = {}
+---@param bufnr (integer) Buffer number, or `0` for current buffer
+---@param client_id (integer) The ID of the |vim.lsp.Client|
+---@param debounce? (integer) (default: 200): Debounce token requests
+---        to the server by the given number in milliseconds
+function M._start(bufnr, client_id, debounce)
+  local highlighter = STHighlighter.active[bufnr]
+
+  if not highlighter then
+    highlighter = STHighlighter:new(bufnr)
+    highlighter.debounce = debounce or 200
+  else
+    highlighter.debounce = debounce or 200
+  end
```

Comment:
The original code used a `max(current_value, debounce or 200)` which was originally intended to make it so that later calls to `start()` that didn't have a debounce parameter passed wouldn't override the debounce value that was set previously.

What might be better is in the `else`, to do something like:

```lua
highlighter.debounce = debounce or higlighter.debounce or 200
```